prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+var languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log('languages array', languages);
+
+var firstElement = languages[0];
+console.log('1st element of array is', firstElement);
+
+var thirdElement = languages[2];
+console.log('3rd element of array is', thirdElement);
+
+var length = languages.length;
+console.log('length of array is', length);
+
+var lastElement = length - 1;
```

Comment:
You need to use the array to get its last element:

```JAVASCRIPT
var lastElement = languages[length - 1];
```

Reply:
makes sense!